gwctl/cmd/get.go

```diff
@@ -102,6 +103,14 @@ func runGet(cmd *cobra.Command, args []string, params *utils.CmdParams) {
 		}
 		gwcPrinter.Print(resourceModel)
 
+	case "namespace", "namespaces":
+		resourceModel, err := discoverer.DiscoverResourcesForNamespace(resourcediscovery.Filter{})
+		if err != nil {
+			fmt.Fprintf(os.Stderr, "failed to discover Namespace resources: %v\n", err)
+			os.Exit(1)
+		}
+		nsPrinter.PrintDescribeView(resourceModel)
```

Comment:
I believe there's a typo here...doesn't this need to be `Print` instead of `PrintDescribeView`?